docs/reference-docs/templates.md

```diff
@@ -0,0 +1,48 @@
+# Templates
+
+A template is a `<filename>.ext.tpl` in a stencil module. 
```

Comment:
```suggestion
A template is a `<filename>.<extension>.tpl` in a stencil module. 
```
i think thats more clear. could be interpreted that `.ext.tpl` is always necessary the other way